### DISCUSSION TITLE
图形化UI开发

### DISCUSSION THREAD

**Discussion by hackyinge** [aoguai/LiYing#2] [Ideas]
如果能提供一个图形化的UI操作就更好了。比如使用 gradio  https://www.gradio.app/  ，或者ComfyUI，都很不错。期望测试跨平台效果

**Comment by aoguai** [comment]
项目一开始是为了帮助我的父母，所以打算最大保留兼容较多的机器避免引入更多依赖（很多新功能在Windows 7无法运行），同时我能力有限不太了解 gradio，所以这可能会开发也可能不会，这暂时不是很重要的功能，当然欢迎提交pr。

**Comment by hackyinge** [comment]
如果你允许，我可以和你共建，把这个项目做成线上可以部署的，打开网页就能用



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月7日(周三) 下午2:04
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [aoguai/LiYing] 图形化UI开发 (Discussion #2)




 
项目一开始是为了帮助我的父母，所以打算最大保留兼容较多的机器避免引入更多依赖（很多新功能在Windows 7无法运行），同时我能力有限不太了解 gradio，所以这可能会开发也可能不会，这暂时不是很重要的功能，当然欢迎提交pr。

**Reply by aoguai** [reply]
如果你可以保证它至少兼容 Windows7 SP及以上，且不影响整合包制作，我欢迎您的符合[贡献指南](https://github.com/aoguai/LiYing/blob/master/docs/CONTRIBUTING.md)的相关贡献

**Comment by hackyinge** [comment]
好的



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月7日(周三) 下午2:18
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [aoguai/LiYing] 图形化UI开发 (Discussion #2)




 
如果你可以保证它至少兼容 Windows7 SP及以上，且不影响整合包制作，我欢迎您的符合贡献指南的相关贡献

**Comment by aoguai** [comment]
https://github.com/aoguai/LiYing/tree/v3.0.0
